src/actions/config.js

```diff
@@ -107,8 +114,9 @@ export function loadConfig() {
     dispatch(configLoading());
 
     try {
+      const configUrl = getConfig();
```

Comment:
If you don't mind, I think it would be good to add something like:
```js
console.log(`Netlify CMS using config file: "${ configURL }"`);
``` 
That way if a config file isn't being recognized and is defaulting to `config.yml`, at least the dev knows that is happening.